src/OctopusPuppet.Gui/ViewModels/DeploymentPlannerViewModel.cs

```diff
@@ -180,38 +184,20 @@ public bool CanCancelDeploy
             }
         }
 
-        private string _octopusApiKey;
-        public string OctopusApiKey
+        private OctopusApiSettings _apiSettings;
+        public OctopusApiSettings ApiSettings
```

Comment:
I merged two properties "OctopusUrl" and "OctopusApiKey" to one "ApiSettings", to call "GetBranchesAndEnvironments" only once.